### PR TITLE
api/rawtx endpoint added to return a JSON containing the raw transaction...

### PR DESCRIPTION
**Pull request by rhoop**
... serialized as a hex-encoded string

### CODE REVIEW

File: app/controllers/transactions.js [hunk 1]
```diff
@@ -34,6 +40,17 @@ exports.send = function(req, res) {
   });
 };
 
+exports.rawTransaction = function (req, res, next, txid) {
+  console.log(txid);
```
Comment (maraoz) [inline app/controllers/transactions.js]: please remove the logging

**Comment by maraoz** [comment]
LGTM sans comment

**Comment by rhoop** [comment]
Comment removed.

**Comment by rhoop** [comment]
Any update on this?

**Comment by maraoz** [comment]
I was waiting for another review to merge, LGTM but we usually want at least 2 devs looking at the changes. ping @matiu @cmgustavo @eordano

File: README.md [hunk 2]
```diff
@@ -180,6 +180,8 @@ The end-points are:
 ```
   /api/tx/[:txid]
   /api/tx/525de308971eabd941b139f46c7198b5af9479325c2395db7f2fb5ae8562556c
+  /api/raw/[:rawid]
```
Comment (fanatid) [inline README.md]: not match with route (`/api/rawtx/:txid`)

**Comment by fanatid** [comment]
May be instead `/api/rawtx/:txid` use `/api/tx/:txid/hex` like chain.com?

**Comment by eordano** [comment]
`/rawtx/:txid` matches blockchain.info's API. There could be an alias there, though. The PR looked good enough to merge it

**Pull request merged**